ferrocene/doc/requirements/src/self-test.rst

```diff
@@ -31,7 +29,16 @@ When executed with ``-vV`` command-line options, the following apply:
 
 - The output of the executable should be UTF-8 text.
 
-- The output of the executable should have at least 3 values: host, commit-hash, and release.
+- The output of the executable should have at least 3 key-value pairs,
+  with these 3 keys: ``host``, ``commit-hash``, and ``release``.
+
+  Example output for rustc:
+
+  .. code-block:: text
+
+     commit-hash: f228f60438264270168a59eba2bc4aaca0ca7c02
+     host: x86_64-unknown-linux-gnu
+     release: 1.81.0-nightly
```

Comment:
I am not sure if we should add examples? I am not saying that we should not, just wondering. As far as I see this is the only instance we do this so far.

Reply:
15285f3588d69bdcc5606e4633765c0efd6af9d4